### ISSUE TITLE
Update app icon badge on push notification

### ISSUE THREAD

**Issue by alderas** [fechanique/cordova-plugin-fcm#385]
Hi,

TL;DR
Is setting app icon badge count on push notification a planned feature in cordova-plugin-fcm?

We have an app with a chat client running Ionic framework and use cordova-fcm-plugin for push notifications. We are looking for a neat solution that sets app icon badge at push notification, i.e. sends the badge count as payload and sets it without need for cordova background mode.  

We've tried to set app icon badge by using a combination of cordova-background-mode and cordova-badge-plugin. However, this turns out to be draining the battery and moreover Apple will not allow our app in app store using this particular solution. I think this a missing piece of an otherwise great library. Phonegap push plugin had this feature at some point, I hope you plan to implement it.

Best regards,

Simon

**Comment by ProximaHaiz** [comment]
Hi. Any updates with this?
